src/Bicep.LangServer/Handlers/BicepDocumentSymbolHandler.cs

```diff
@@ -76,60 +76,32 @@ private DocumentSymbol CreateDocumentSymbol(SemanticModel model, DeclaredSymbol
         }
 
 
-        private SymbolKind SelectSymbolKind(DeclaredSymbol symbol)
+        private static SymbolKind SelectSymbolKind(DeclaredSymbol symbol) => symbol switch
         {
-            switch (symbol)
-            {
-                case ImportedNamespaceSymbol _:
-                    return SymbolKind.Namespace;
-
-                case ParameterSymbol _:
-                    return SymbolKind.Field;
-
-                case VariableSymbol _:
-                    return SymbolKind.Variable;
-
-                case ResourceSymbol resource:
-                    return SymbolKind.Object;
-
-                case ModuleSymbol module:
-                    return SymbolKind.Module;
-
-                case OutputSymbol output:
-                    return SymbolKind.Interface;
-
-                default:
-                    return SymbolKind.Key;
-            }
-        }
+            ImportedNamespaceSymbol => SymbolKind.Namespace,
+            ParameterSymbol => SymbolKind.Field,
+            VariableSymbol => SymbolKind.Variable,
+            ResourceSymbol => SymbolKind.Object,
+            ModuleSymbol => SymbolKind.Module,
+            OutputSymbol => SymbolKind.Interface,
+            ParameterAssignmentSymbol => SymbolKind.Constant,
```

Comment:
>ParameterAssignmentSymbol [](http://example.com/codeflow?start=12&length=25)

This line and the one commented below is the actual change. Everything else is just a refactor.